portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -20,28 +20,35 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.sps.data.Comment;
+import java.lang.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 
 @WebServlet("/list-post")
 public class ListCommentServlet extends HttpServlet {
+    private ArrayList<Comment> comments = new ArrayList<Comment>();
```

Comment:
since you aren't using anything special with ArrayList implementation, it would be better to use the generic type "List", so you can do "List<Comment> comments = new ArrayList<>()" (also notice on right-hand side, you don't need to specify <Comment> again, you can just leave as <> since it is inferred from the left-hand side;

Reply:
Thank you for the tip again I will make those changes.

Reply:
edit: "List<Comment> comments = new ArrayList<>()"